id-library-showcase/Cargo.toml

```diff
@@ -8,6 +8,8 @@ license-file = "../LICENSE"
 
 [dependencies]
 concordium-rust-sdk = { version = "*", path = "../deps/concordium-rust-sdk/"}
+# Add base directly to enable the encryption feature.
```

Comment:
Should we move this feature into Rust instead? Or is it unrealistic for developers to want to use this as well?

Reply:
I assume you mean the Rust-SDK, and I am not sure. What do you think @abizjak ?